src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
```

Comment:
In order not to duplicate this entry, it is necessary to add it to reset.css:
`* {`
`  font-family: inherit;`
`}`